AUTHORS

```diff
@@ -682,6 +682,7 @@ answer newbie questions, and generally made Django that much better:
     Max Derkachev <mderk@yandex.ru>
     Max Smolens <msmolens@gmail.com>
     Maxime Lorant <maxime.lorant@gmail.com>
+    Maxime Toussaint <m.toussaint@mail.com>
```

Comment:
I could be completely wrong so feel free to dismiss but given the popularity of GMail was this meant to be `@gmail.com`?

Reply:
It was not a typo, I use a system that is not very well known